.claude/tasks/2025-08-27-nix-devenv-setup.md

```
@@ -0,0 +1,136 @@
+  - [ ] nil - Nix language server diagnostics
+- [ ] Configure Haskell hooks:
+  - [ ] fourmolu formatting enforcement
+- [ ] Configure TypeScript/JavaScript hooks:
+  - [ ] prettier formatting for JS/TS/JSON/YAML
+  - [ ] eslint for code quality checks
+- [ ] Add yamlfmt for YAML file formatting
+- [ ] Add taplo for TOML file formatting and linting
+- [ ] Test pre-commit hook installation and execution
+
+### Section 5: Direnv Integration
+- [ ] Create `.envrc` file with `use flake` directive
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)